### DISCUSSION TITLE
Hyper 1.0 Examples

### DISCUSSION THREAD

**Discussion by hariria** [tokio-rs/axum#2432] [Q&A] [answered]
### Summary

Hi, I was looking to try and benchmark Axum and was looking into some of the examples but noticed that several of them were outdated and that there hasn't been much activity to upgrade them to Hyper 1.0. Is there a timeline for when they might upgrade?

### axum version

0.7.2

**Comment by davidpdrsn** [comment, marked as answer]
There is no timeline but you can watch https://github.com/tokio-rs/axum/issues/2356. Feel free to help updating stuff.
